### DISCUSSION TITLE
Podman Deskop: initialization takes forever

### DISCUSSION THREAD

**Discussion by ImperrattoR** [containers/podman#20248] [Q&A] [answered]
Hello,

I have successfully installed Docker Podman. Everything went smooth, and everything was green.
But when I try to initialize the podman it takes forever.

I have successfully installed Docker Podman. Everything went smoothly, and everything was 'green'. However, when I try to initialize Podman, it takes forever.

Logs:
`Container connections: 0 (0 running)
Logs

MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 extensions-started listeners added. Use emitter.setMaxListeners() to increase limit
main ↪️ PluginSystem: received dom-ready event from the UI
main ↪️ Delayed startup, flushing 7 events
main ↪️ System ready. Loading extensions...
main ↪️ Activating extension (podman-desktop.compose) with max activation time of 10 seconds
main ↪️ Activating extension (podman-desktop.docker) with max activation time of 10 seconds
main ↪️ Activating extension (podman-desktop.kind) with max activation time of 10 seconds
main ↪️ Activating extension (podman-desktop.kube-context) with max activation time of 10 seconds
main ↪️ [kube-context] starting extension kube-context
main ↪️ Activating extension (podman-desktop.lima) with max activation time of 10 seconds
main ↪️ [lima] Could not find socket at C:\Users\####\.lima\podman\sock\podman.sock
main ↪️ Activating extension (podman-desktop.podman) with max activation time of 10 seconds
main ↪️ Activating extension (podman-desktop.registries) with max activation time of 10 seconds
main ↪️ Activating extension (podman-desktop.compose) ended in 91 milliseconds
main ↪️ Activating extension (podman-desktop.docker) ended in 88 milliseconds
main ↪️ Activating extension (podman-desktop.lima) ended in 22 milliseconds
main ↪️ Activating extension (podman-desktop.kube-context) ended in 105 milliseconds
main ↪️ Activating extension (podman-desktop.registries) ended in 83 milliseconds
main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 216.82590001821518ms
main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 387.1599000096321ms
main ↪️ [podman] Error while pinging docker as podman Error: connect ENOENT //./pipe/docker_engine
main ↪️ [kind] [object Object]
main ↪️ Activating extension (podman-desktop.kind) ended in 752 milliseconds
main ↪️ Activating extension (podman-desktop.podman) ended in 769 milliseconds
main ↪️ PluginSystem: initialization done.
main ↪️ Autostarting container engine
main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 186.90650004148483ms
main ↪️ Error while fetching API groups: Error: connect ETIMEDOUT 192.168.99.100:8443
main ↪️ Error while fetching API groups: Error: connect ETIMEDOUT 192.168.99.100:8443`

After that, I attempted to create a Podman machine in Podman Desktop, but I encountered the following error:
`Extracting compressed file

Importing operating system into WSL (this may take a few minutes on a new WSL install)...

A distribution with the supplied name already exists.

Error: the WSL import of guest OS failed: exit status 0xffffffff

Command execution failed with exit code 125
`

Is anyone else experiencing the same problem, or does anyone know how to fix it?

Best regards

**Comment by vrothberg** [comment]
Thanks for reaching out, Mike!  Can you share which version of Podman and Podman Desktop you are using?

@benoitf have you seen this issue before?

**Reply by benoitf** [reply, marked as answer]
I think `Error: the WSL import of guest OS failed: exit status 0xffffffff` is when some files have been deleted but we try to install again Podman

maybe try command
`wsl --unregister podman-machine-default`

before trying to initialize the podman machine again

**Reply by ImperrattoR** [reply]
Thanks, it worked for me.

**Reply by vrothberg** [reply]
@n1hility anything we can do on the Podman-side to improve on that?  Maybe Podman can auto-fix or improve the error message?

**Comment by ImperrattoR** [comment]
I'm using Podman version 4.6.2 and Podman Desktop version 1.4.0.

I have just recently installed Podman Desktop, so I haven't encountered this issue before because Podman never worked for me previously.
